lib/cocoapods-playgrounds/workspace.rb

```diff
@@ -168,8 +169,8 @@ def generate_project
     def generate_swift_code(path)
       File.open(path + 'Contents.swift', 'w') do |f|
         f.write("//: Please build the scheme '#{target_name}' first\n")
-        f.write("import XCPlayground\n")
```

Comment:
Yes definitely. I only made this as a quick fix. I will update to include some swift version #ifdefs.